tests/plugins/imagebase/features/manual/linkcontextmenu.md

```diff
@@ -1,6 +1,7 @@
 @bender-tags: 4.9.0, feature, 932
 @bender-ui: collapsed
 @bender-ckeditor-plugins: sourcearea, wysiwygarea, floatingspace, toolbar, imagebase, link, htmlwriter, elementspath, contextmenu
+@bender-include: %BASE_PATH%/plugins/easyimage/manual/_helpers/tools.js
```

Comment:
`tests/plugins/easyimage/manual/_helpers/tools.js` is **already loaded** in the HTML file.

Reply:
I assume you meant tests:
- `tests/plugins/imagebase/features/manual/uplaod`
- `tests/plugins/imagebase/features/manual/uploadsimple`

`tests/plugins/imagebase/features/manual/linkcontextmenu.md` has loaded library only once in markdown file.